airbyte-integrations/bases/source-acceptance-test/unit_tests/test_utils.py

```diff
@@ -259,12 +259,12 @@ def test_failed_reading(traceback, container_error, last_line, expected_error):
     "command,wait_timeout,expected_count",
     (
         (
-            "cnt=0; while [ $cnt -lt 10 ]; do cnt=$((cnt+1)); echo something; done",
+            "cnt=0; while [ $cnt -lt 10 ]; do cnt=$((cnt+1)); sleep 0.5; echo something; done",
             0,
             10,
         ),
         # Sometimes a container can finish own work before python tries to read it
-        ("echo something;", 0.1, 1),
+        ("echo something;", 3, 1),
     ),
     ids=["standard", "waiting"],
```

Comment:
Both the `standard` and `waiting` test can fail:
- `standard`: https://github.com/airbytehq/airbyte/runs/4735244889?check_suite_focus=true
- `waiting`: https://github.com/airbytehq/airbyte/runs/4731157302?check_suite_focus=true

So they are both reverted.